grammar.js

```diff
@@ -187,8 +189,8 @@ module.exports = grammar({
 
     module: $ => seq(
       'module',
-      $.constant,
-      $._terminator,
+      choice($.constant, $.scope_resolution),
+      choice(';', $._line_break),
```

Comment:
:+1: Oh, great catch; I just did this in an attempt to address a conflict that ended up being unrelated. There's no need for me to make this change.